app/assets/javascripts/application.js

```diff
@@ -34,10 +34,10 @@ function isShortHeightScreen() {
   return $(window).height() < 768 && !isMobileResolution();
 }
 
-window.setTimeout(function () {
+window.setTimeout(function() {
   // When the user is given an error message, we should not auto-hide it so that
   // they can fully read it and potentially copy/paste it into an issue.
-  $(".alert").fadeTo(1000, 0).slideUp(1000, function () {
+  $(".alert").not(".error").fadeTo(1000, 0).slideUp(1000, function(){
```

Comment:
nitpicks, but I like a a space between `()` and the opening `{` of a function in Javascript.